lib/modules/dosomething/dosomething_campaign/includes/Campaign.php

```diff
@@ -376,8 +376,12 @@ protected function getMobileAppDate() {
       $timing = [];
       $timing['dates'] = (dosomething_helpers_extract_field_data($this->node->field_mobile_app_date));
 
-      foreach ($timing['dates'] as $key => $date) {
-        $timing['dates'][$key] = dosomething_helpers_convert_date($date);
+      if ($timing['dates'] != null) {
+        foreach ($timing['dates'] as $key => $date) {
+          $timing['dates'][$key] = dosomething_helpers_convert_date($date);
+        }
+      } else {
+        $timing['dates'] === null;
```

Comment:
when you're assigning a var, one `=` will do 
do you need this though, if it isn't set shouldn't it already be null?